lang/cs/Org.Apache.REEF.Client/AzureBatch/Parameters/AzureBatchPoolDriverPortsList.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+using System.Collections.Generic;
+using Org.Apache.REEF.Tang.Annotations;
+
+namespace Org.Apache.REEF.Client.AzureBatch.Parameters
+{
+    [NamedParameter(Documentation = "The Azure Batch Pool Driver Http Server Ports List")]
+    public sealed class AzureBatchPoolDriverPortsList : Name<IList<string>>
```

Comment:
>string [](start = 67, length = 6)

Encountered this when looking at my related changes. Why this is not IList<int> instead?

Reply:
The reason was from the example in TestListInjection.java, to bind a List<Integer> , it requires to use List<String>

---
In reply to: [195608449](https://github.com/apache/reef/pull/1468#discussion_r195608449) [](ancestors = 195608449)

Reply:
By the way, it seems binding List<int> is not supported.

---
In reply to: [195804156](https://github.com/apache/reef/pull/1468#discussion_r195804156) [](ancestors = 195804156,195608449)